### PR TITLE
[MP-245] Add backend validation for minting

### PR DESCRIPTION
**Pull request by sidorovkirill**

### CODE REVIEW

File: src/services/MintTrackerService.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+const heroRepository = require("../repositories/HeroesRepository");
+const userRepository = require("../repositories/UsersRepository");
+
+const MINT_EVENT_NAME = "TokenMintedWithSignature";
+const LAST_BLOCK_FIELD_NAME = "lastBlock";
+
+class MintTrackerService {
+  async startTracking() {
+    await this.getPastEvent();
+    // await this.subscribeToEvents();
```
Comment (Vukiz) [inline src/services/MintTrackerService.js:12]: Why is it commented?

File: src/services/MintTrackerService.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    }
+  }
+
+  async handleEvent(event) {
+    const tokenId = event.returnValues.tokenId
+    const heroId = event.returnValues.backendId
+    if (tokenId && heroId) {
+      await heroRepository.updateTokenId(heroId, tokenId);
+    }
+    this.setLastBlock(event.blockNumber);
```
Comment (Vukiz) [inline src/services/MintTrackerService.js:44]: Doesn't sound right, what if we missed the block and manually update it? The current last block can be newer than the one we are trying to handle?

**Review by Vukiz — CHANGES_REQUESTED** [review]

File: src/services/MintTrackerService.js (hunk 2 above)
Comment (sidorovkirill) [reply src/services/MintTrackerService.js:44]: There is no way to update the last block manually.

File: src/services/MintTrackerService.js (hunk 1 above)
Comment (sidorovkirill) [reply src/services/MintTrackerService.js:12]: Because we can't test it without WebSocket connect, I still didn't get access to WebSocket rpc.

**Review by Vukiz — APPROVED** [review]

**Pull request merged**